src/js/components/CharityPage.jsx

```diff
@@ -56,24 +56,41 @@ const CharityPage = () => {
 	const impactColumn = (
 		<Col md="7" xs="12" className="column impact-column">
 			<div className="header">
+				{charity.logo? <div><img src={charity.logo} alt='Charity logo' className='charity-logo'/></div> : null}
 				<h1 className="charity-name">
 					{charity.displayName || charity.name} <small><EditLink charity={charity} /></small>
 				</h1>
-				<CharityTags className="why-tags" tagsString={charity.whyTags} />
-				<CharityTags className="where-tags" tagsString={charity.whereTags} />
+				<div className='description-short'>
+					{charity.summaryDescription? <MDText source={charity.summaryDescription} /> : null}
+				</div>
+				<CharityTags whyTagsString={charity.whyTags} whereTagsString={charity.whereTags} />
 			</div>
 			<CharityDonate charity={charity} />
+			<div className="header">
+				<h2 className="charity-section-title">
+					About {charity.displayName || charity.name}
+				</h2>
+			</div>
+			<div className='charity-section-text'>
+				<div className='description-short'>
+					{charity.summaryDescription? <MDText source={charity.summaryDescription} /> : null}
+				</div>
+				<div className='description-long'>
+					{charity.description? <MDText source={charity.description} /> : null}
+				</div>
+			</div>
+			<div className="header">
+				<h2 className="charity-section-title">Our Analysis</h2>
```

Comment:
Handle the `! charity.recommendation` case.

Reply:
Done. (No 'Our Analysis' title shown if there is no analysis to show)